### PR TITLE
Fixes #628 : Converts merge_badges into class file

### LINKED ISSUE #628: Convert merge_badges.py into a class
Presently os.system is used to execute merge_badges because merge_badges does have a proper interface/api to use it from another file. It is a standalone script. This was alright when badgeyay was a command line tool to create badges. However now, using os.system to run the script seems quite hacky and limits its use. Instead, merge_badges.py should be converted to a proper module containing a class
which can be imported to main.py and used. I guess this is the proper workflow rather than executing it as a stand alone script.

### PR DESCRIPTION
**Pull request by gabru-md**
Creates a new file for merge_badges_class.py
so that it can be imported. Changes have been made to main.py as well.

Cheers!

Fixes #628 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

### Preview Link 

- **Replace XXX with your PR no**
- Link to live demo: http://pr-650-fossasia-badgeyay.surge.sh  

#### Changes proposed in this pull request:

- Creates `merge_badges_class.py`
- Changes `main.py` for the same

### CODE REVIEW

**Comment by gabru-md** [comment]
CC : @yashLadha @ParthS007 @djmgit 
Please review

File: backend/app/merge_badges_class.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import os
+from generate_badges import GenerateBadges
+from cairosvg import svg2pdf
+from PyPDF2 import PdfFileMerger
+
+
+class MergeBadges:
+    """docstring for MergeBadges"""
+    APP_ROOT = os.path.dirname(os.path.abspath(__file__))
```
Comment (yashLadha) [inline backend/app/merge_badges_class.py]: Switch the properties to self.  Also same elsewhere

**Comment by yashLadha** [comment]
@djmgit Hound is not using the config specified in the code. Ref: #655 Will solve the hound errors.

**Comment by djmgit** [comment]
@yashLadha alright, got it :+1:

**Comment by gabru-md** [comment]
@djmgit @yashLadha @ParthS007 Please review now. I've added them into `__init__` now.
The only thing failing right now is the line length i suppose :sweat_smile:

**Comment by djmgit** [comment]
class

generatePDFs is causing trouble I guess.

**Comment by gabru-md** [comment]
@djmgit @ParthS007 merge this now? It passes Travis and Codacy as well. :D

**Comment by gabru-md** [comment]
Merge please?

**Pull request merged**